trillian/migrillian/election/election.go

```diff
@@ -32,12 +32,13 @@ type Election interface {
 	// Observe returns a "mastership context" which remains active until the
 	// instance stops being the master, or the passed in context is canceled. If
 	// the instance is not the master during this call, returns an outright
-	// canceled context.
+	// canceled context. In particular, this will happen if Observe is called
```

Comment:
Maybe "already-canceled" rather than "outright canceled" ?

Also, I think your existing text probably already covered the no-preceding-Await case fine, I just didn't read it properly :-)

Reply:
Changed to "already", thanks.

I left the no-preceding-Await clause in case anyone else doesn't read it properly :)